client/src/pages/dashboard/NewEventType.js

```diff
@@ -17,122 +19,164 @@ import useMediaQuery from '@material-ui/core/useMediaQuery';
 
 
 const NewEventType = () => {
-    const theme = useTheme();
-    const matches = useMediaQuery(theme.breakpoints.up("sm"));
-    return (
-        <Container>
+  const theme = useTheme();
+  const matches = useMediaQuery(theme.breakpoints.up("sm"));
+  
+    //   Submit Form Data to Back-End
+
+    const userId = useContext(UserProvider.context).user.id;
```

Comment:
```suggestion
    const userContext = useContext(UserProvider.context);
```

so when you hit `handleSubmit`, you can add a variable `const userId = userContext.user.id` should work inside that function

Reply:
thanks, bonnie. that change worked. I will commit this and close the PR.